pennylane/gradients/jvp.py

```diff
@@ -179,8 +179,8 @@ def compute_jvp_single(tangent, jac):
         jac = qml.math.reshape(jac, new_shape)
         return qml.math.tensordot(jac, tangent, [[-1], [0]])
 
-    tangent_ndims = [t.ndim for t in tangent]
-    if isinstance(tangent, (tuple, list)) and any(t.ndim > 0 for t in tangent):
+    tangent_ndims = [getattr(t, "ndim", 0) for t in tangent]
+    if isinstance(tangent, (tuple, list)) and any(getattr(t, "ndim", 0) > 0 for t in tangent):
```

Comment:
```suggestion
    if isinstance(tangent, (tuple, list)) and any(t > 0 for t in tangent_ndims):
```

Reply:
```suggestion
    if isinstance(tangent, (tuple, list)) and any(ndim > 0 for ndim in tangent_ndims):
```

Reply:
(Same as @timmysilv's suggestion, but with `ndim` instead of `t`.